src/test/java/org/fundacionjala/coding/rodrigo/movierefactor/RentalTest.java

```diff
@@ -0,0 +1,66 @@
+package org.fundacionjala.coding.rodrigo.movierefactor;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+/**
+ * Test for the Rental class.
+ */
+public class RentalTest {
+    private Customer customer;
+    private Movie movie1;
+    private Movie movie2;
+    private Movie movie3;
+    private Movie movie4;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "movie4" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)